File: Linguagem C++/glicose.cpp
#include <iostream>

using namespace std;

int main(){

    double glicose;

    cout << "Digite a medida da glicose: ";
    cin >> glicose;
    if (glicose <= 100) {
        cout << "Classificacao: normal ";
    }
    else if (glicose <= 140)  {
        cout << "Classificacao: elevada ";
    }
    else {
        cout << "Classificacao: diabetes ";
    }


    return 0;
}

File: Linguagem Python/EXs - Estrutura Condicional PY/glicose.py
glicose = float(input("Digite a medida da glicose: "))
if glicose <= 100:
    classificacao = "Normal"
elif glicose <= 140:
    classificacao = "Elevado"
else:
    classificacao = "Diabets"

print(f"Classificacao = {classificacao}")